components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import { action } from "@storybook/addon-actions";
```

Comment:
`@` is supposed to be above "r".

```suggestion
import { action } from "@storybook/addon-actions";
import * as React from "react";
```